server/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "target": "es2017",
     "outDir": "./dist",
+    "baseUrl": "./",
```

Comment:
unfortunately, our current setup is not compatible with this change. If you create a production Docker image it won't work.

Reply:
thx fixed